### PR TITLE
feat: leader profiles block

### LINKED ISSUE #37: [Blocks] Implement Leader Profile block
Original: https://www.netcentric.biz/who-we-are.html
Franklin: https://docs.google.com/document/d/18v60MnckEhak1ykRMH10FdcapkntIQSKxQQX_cJWJEw/edit#

### PR DESCRIPTION
**Pull request by buuhuu**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #37 

Test URLs:
- Before: https://main--netcentric--hlxsites.hlx.page/who-we-are
- After: https://37-leader-profiles--netcentric--hlxsites.hlx.page/who-we-are

### CODE REVIEW

**Comment by hannessolo** [comment]
Can you link the page (in the before/after) where the leader block is meant to appear?

**Comment by mhaack** [comment]
@hannessolo done.

**Comment by mhaack** [comment]
@Buuhuu not sure if this is already read for review since you added us. Looks good, but people details don't show up. So I assume it is still work in progress.

**Review by mhaack — APPROVED** [review]

File: blocks/leader-profiles/leader-profiles.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
+.leader-profiles>ul {
```
Comment (hannessolo) [inline blocks/leader-profiles/leader-profiles.css:1]: Can you set `cursor: pointer` so its visually indicated that you can click the cards?

**Review by hannessolo — CHANGES_REQUESTED** [review]
LGTM except for cursor

**Review by hannessolo — APPROVED** [review]

**Pull request merged**